pkg/jx/cmd/step_create_task.go

```diff
@@ -68,6 +69,7 @@ type StepCreateTaskOptions struct {
 	DeleteTempDir  bool
 	ViewSteps      bool
 	Duration       time.Duration
+	FromYaml       bool
```

Comment:
This is a bad name for sure, so any suggestions are welcome.

Reply:
`FromRepo` maybe?

Reply:
Ok, I'll use FromRepo for now, we can adjust as needed later!